lib/taskana-core/src/test/java/acceptance/jobs/helper/SqlConnectionRunnerAccTest.java

```diff
@@ -0,0 +1,62 @@
+package acceptance.jobs.helper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import acceptance.AbstractAccTest;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.common.api.exceptions.SystemException;
+import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.task.internal.jobs.helper.SqlConnectionRunner;
+
+/** Acceptance test for all "jobs tasks runner" scenarios. */
+@ExtendWith(JaasExtension.class)
+class SqlConnectionRunnerAccTest extends AbstractAccTest {
+
+  @BeforeEach
+  void before() throws Exception {
+    // required if single tests modify database
+    // TODO split test class into readOnly & modifying tests to improve performance
+    resetDb(false);
+  }
+
+  @Test
+  void should_executeSimpleQuery() {
```

Comment:
in theory our testing guidelines (https://taskana.atlassian.net/wiki/spaces/TAS/pages/2634055697/Testing+Guidelines) say to provide a 'when' or a 'for' case, but i like this approach for simple tests
Maybe this could be a topic for our COP?

Reply:
i have nothing else to add for "when" or "for" ... yeah maybe let's discuss that in an cop item

Reply:
Would you be so kind and create this item?

Reply:
Done